modules/nf-core/modules/bcftools/norm/main.nf

```diff
@@ -0,0 +1,45 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process BCFTOOLS_NORM {
+    tag "$meta.id"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::bcftools=1.13" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/bcftools:1.13--h3a49de5_0"
+    } else {
+        container "quay.io/biocontainers/bcftools:1.13--h3a49de5_0"
+    }
+
+    input:
+    tuple val(meta), path(vcf)
+    path(fasta)
+
+    output:
+    tuple val(meta), path("*.gz") , emit: vcf
+    path "versions.yml"           , emit: version
+
+    script:
+    def software = getSoftwareName(task.process)
+    def prefix   = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
+    """
+    bcftools norm \\
+        --fasta-ref ${fasta} \\
+        --output ${prefix}.vcf.gz \\
+        $options.args \\
+        --threads $task.cpus \\
+        ${vcf}
```

Comment:
Considering the way the module is written I think it has to include include the option `--output-type z`. This option seem to be an underlying requirement as the module can only emit gzipped vcf:s. But I might be missing something :)